package.json

```diff
@@ -105,7 +107,9 @@
     "commit": "git-cz",
     "cover": "istanbul cover test/index.js",
     "lint": "standard && tslint -c tslint.json ./src/**/*.ts ./test/**/*.ts",
-    "test": "npm run lint && ts-node test | tap-diff",
+    "sinopia": "sinopia -c ./sinopia/config.yaml",
+    "test:e2e": "ts-node test | tap-diff",
+    "test": "npm run lint && npm-run-all -p -r sinopia test:e2e",
```

Comment:
"test": "npm-run-all lint -p -r sinopia test:e2e",
regarding docs https://github.com/mysticatea/npm-run-all/blob/master/docs/npm-run-all.md#run-a-mix-of-sequential-and-parallel-scripts